File: SWIFT/Cap04/c04ex11.swift
// c04ex1.swift

import Foundation

var N, R4, R5: Int

print("Entre um numero inteiro: ", terminator: "")
if let inputN = readLine(), let n = Int(inputN) {
    N = n
} else {
    fatalError("Entrada invalida.")
}

R4 = N % 4
R5 = N % 5

if (R4 == 0 && R5 == 0) {
    print("Resultado = \(N)")
} else {
    print("Valor nao e divisivel por 4 e 5")
}

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: DART/Cap04/c04ex11.dart
// c04ex11.dart

import 'dart:io';

void main() {
  int N;
  int R4;
  int R5;

  stdout.write("Entre um numero inteiro: ");
  N = int.tryParse(stdin.readLineSync() ?? '') ?? 0;

  R4 = N % 4;
  R5 = N % 5;

  if (R4 == 0 && R5 == 0) {
    print("Resultado = $N");
  } else {
    print("Valor nao e divisivel por 4 e 5");
  }

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
